io/file/fileutil.go

```diff
@@ -183,6 +183,40 @@ func RecursiveFileFind(filename, dir string) (bool, string, error) {
 	return found, fpath, nil
 }
 
+// RecursiveDirFind searches for directory in a directory and its subdirectories.
+func RecursiveDirFind(dirname, dir string) (bool, string, error) {
```

Comment:
maybe I should use this for the keystore import, mine only goes 2 levels deep...